package.json

```diff
@@ -27,12 +27,14 @@
   },
   "license": "MIT",
   "dependencies": {
+    "@aws-sdk/client-dynamodb": "^3.0.0",
+    "@aws-sdk/lib-dynamodb": "^3.0.0",
+    "@aws-sdk/util-buffer-from": "^3.0.0",
```

Comment:
Should these remain peer dependencies?

My main concern is API upgrading one of these, and then passing in some object/client which has new behaviour to a dynogels constructor.

Reply:
Changed to peer dependency. This allows us to fail in cases where the library user has non-matching aws version (e.g. version 4 or 2).